### PR TITLE
updates for dashboard

### PR DESCRIPTION
**Pull request by 0xSmick**

### CODE REVIEW

**Comment by 0xlarry** [comment]
looks good, you could consolidate to a single function and get historic is just based on the params inputted

**Comment by nardevvv** [comment]
> looks good, you could consolidate to a single function and get historic is just based on the params inputted

^ Yup, can just use as one function

File: src/index.ts [hunk 1]
```diff
@@ -4,3 +4,4 @@ export * from './classes/request.js';
 export * from './classes/sdk.js';
 export * from './classes/user.js';
 export * from './types/index.js';
+export { TorqueAdminClient } from './classes/admin.js';
```
Comment (nardevvv) [inline src/index.ts:7]: This line can be removed. 

This will take care of it: `export * from './classes/admin.js';`

File: src/classes/admin.ts [hunk 2]
```diff
@@ -128,18 +128,43 @@ export class TorqueAdminClient {
...
+   */
+  public async getHistoricCampaigns(params?: CampaignRequestParams) {
+    const historicParams: CampaignRequestParams = {
+      ...params,
+      status: 'ENDED',
+    };
+    return this.getCampaigns(historicParams, true);
+  }
+
+  private async getCampaigns(params?: CampaignRequestParams, includeHistoric: boolean = false) {
```
Comment (nardevvv) [inline src/classes/admin.ts:150]: After a second look, making this a private method while adding the others as public makes sense

**Comment by nardevvv** [comment]
I can up the version the deploy. The client build will fail until then.

**Pull request merged**